src/application/application_test.ts

```diff
@@ -9,9 +9,9 @@ test('listenEvent() should listen for events and return a stream', (t) => {
   const api = Api('')
   const application = new Application(api);
   const spy = sinon.spy(api.event, 'stream')
-  application.listenEvent({ filter: { instanceHash: '1', key: '2' } })
+  application.listenEvent({ filter: { instanceHash: Buffer.from('1'), key: '2' } })
   const req = spy.getCall(0).args[0]
-  t.equal(req.filter.instanceHash, '1')
+  t.ok(req.filter.instanceHash.equals(Buffer.from('1')))
```

Comment:
I'm not sure but I think the ok is just testing if there is no error thrown

Reply:
```
         * Assert that value is truthy with an optional description message msg.
        ok(value: any, msg?: string): void;
        true(value: any, msg?: string): void;
        assert(value: any, msg?: string): void;
```
i think those function are alias